cfep-03.md

```diff
@@ -0,0 +1,72 @@
+
+<table>
+<tr><td> Title </td><td> Providing manually uploaded builds on the conda-forge anaconda channel</td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Hadrien Mary &lt;hadrien.mary@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Sept 27, 2016</td></tr>
+<tr><td> Updated </td><td> Sept 27, 2016</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Some packages are very hard to build on the currently used CI (CircleCI, Travis and AppVeyor). It's most of the time because of time limitations in the CI website and also because of the complexity of the build.
```

Comment:
@astrofrog that is an important question and the text was not clear. The new text covers only recipes that we can "just download and build." Excluding things that are hard to create recipes but easy to build locally. The reason is to avoid hacks that may obscure the builds and ensure transparency of what will be built locally.

TL;DR this cfep covers only CI build time limitation.